### DISCUSSION TITLE
Not getting updates from add on store

### DISCUSSION THREAD

**Discussion by Substarbo** [zwave-js/zwave-js#4195] [Q&A]
My official add on store disappeared after an update a while ago and now I am unable to get updates for for zwave js. I have just completely lost zwave Js becuse its an old version. Is there any way to get the update with out the add on store?

**Comment by AlCalzone** [comment]
I think you're looking for this repo
=> https://github.com/home-assistant/core

This project is just the Z-Wave driver library.
